src/components/sykmeldte/SykmeldteNotifying/MarkAllAsReadModal.tsx

```diff
@@ -0,0 +1,66 @@
+import { useCallback } from 'react'
+import { useMutation, useQuery } from '@apollo/client'
+import { BodyLong, Button, Heading, Modal } from '@navikt/ds-react'
+import { logger } from '@navikt/next-logger'
+
+import { logAmplitudeEvent } from '../../../amplitude/amplitude'
+import {
+    MarkAllSykmeldingerAndSoknaderAsReadDocument,
+    MineSykmeldteDocument,
+} from '../../../graphql/queries/graphql.generated'
+
+import styles from './MarkAllAsReadModal.module.css'
+
+function MarkAllAsReadModal({ onClose }: { onClose: (wasCancelled: boolean) => void }): JSX.Element {
+    const headingId = 'mark-all-notifications-as-read-modal'
+    const [markAllSykmeldingerAndSoknaderAsRead, { loading }] = useMutation(
+        MarkAllSykmeldingerAndSoknaderAsReadDocument,
+    )
+    const { refetch, loading: refetching } = useQuery(MineSykmeldteDocument)
+
+    const onSuccess = useCallback(async () => {
+        await refetch()
+        onClose(false)
```

Comment:
ville bare flytta disse to som en inline-arrow function på `onComplete`, siden den kun brukes der inne.